mlir/include/Quantum/IR/QuantumOps.td

```diff
@@ -448,7 +448,9 @@ def HamiltonianOp : Observable_Op<"hamiltonian", [Pure]> {
         Variadic<ObservableType>:$terms
     );
 
-    let results = (outs ObservableType:$obs);
+    let results = (outs
```

Comment:
What is changed the code style of `.td` files?

Reply:
No change in the style, this just brings it in line with the existing definitions.